server/src-lib/Hasura/RQL/Types/Column.hs

```diff
@@ -138,52 +146,59 @@ parseTxtEncodedPGValue colTy val =
 -- | “Raw” column info, as stored in the catalog (but not in the schema cache). Instead of
 -- containing a 'PGColumnType', it only contains a 'PGScalarType', which is combined with the
 -- 'pcirReferences' field and other table data to eventually resolve the type to a 'PGColumnType'.
-data PGRawColumnInfo
-  = PGRawColumnInfo
+data RawColumnInfo (b :: Backend)
+  = RawColumnInfo
   { prciName        :: !PGCol
```

Comment:
What about `PGCol`? Shouldn't that be something like `Column b`?

Reply:
Done.